Stripe/STPCardIOProxy.m

```diff
@@ -0,0 +1,129 @@
+//
+//  STPCardIOProxy.m
+//  Stripe
+//
+//  Created by Ben Guo on 5/12/17.
+//  Copyright © 2017 Stripe, Inc. All rights reserved.
+//
+
+#import "STPCardIOProxy.h"
+#import "STPCardParams.h"
+#import "STPAnalyticsClient.h"
+
+@protocol STPClassProxy
```

Comment:
Would it be easier to copy a few of the CardIO headers into our project instead of doing the proxy this way? Then you can just do something like `if ([CardIOUtilities class]) { //foo }` and use the class directly if it exists (the same way we check for Apple class availability on older OS versions).